source/MRViewer/MRObjectTransformWidget.h

```diff
@@ -53,9 +53,22 @@ class MRVIEWER_CLASS ObjectTransformWidget : public MultiListener<MouseDownListe
     void setPickThrough( bool on ) { pickThrough_ = on; }
     bool getPickThrough() const { return pickThrough_; }
 
+    enum AxisTransformMode
+    {
+        Translation,
+        Scaling,
+    };
+    AxisTransformMode getAxisTransformMode() const { return axisTransformMode_; };
+    void setAxisTransformMode( AxisTransformMode mode ) { axisTransformMode_ = mode; };
```

Comment:
please add commets about this structure and how scale works

Reply:
Added.